docs/assets/less/stacks-documentation.less

```diff
@@ -231,6 +231,20 @@
     white-space: normal;
 }
 
+//  ============================================================================
+//  $  CODE COPY BUTTON
+//  ----------------------------------------------------------------------------
+.stacks-copy-btn {
+    .svg-icon,
+    .svg-icon * {
+        pointer-events: none;
+    }
+
+    .iconCheckmark {
+        color: var(--green-500);
+    }
```

Comment:
My normal trick of applying `pointer-events: none` to SVG isn't quite working. When moving the mouse within the button there's weird behavior with the tooltip hiding and showing.